forgerock-auth/src/test/java/org/forgerock/android/auth/callback/NumberAttributeInputCallbackTest.java

```diff
@@ -0,0 +1,165 @@
+                "                    \"value\": true\n" +
+                "                },\n" +
+                "                {\n" +
+                "                    \"name\": \"policies\",\n" +
+                "                    \"value\": {\n" +
+                "                        \"policyRequirements\": [\n" +
+                "                            \"VALID_TYPE\"\n" +
+                "                        ],\n" +
+                "                        \"fallbackPolicies\": null,\n" +
+                "                        \"name\": \"age\",\n" +
+                "                        \"policies\": [\n" +
+                "                            {\n" +
+                "                                \"policyRequirements\": [\n" +
+                "                                    \"VALID_TYPE\"\n" +
+                "                                ],\n" +
+                "                                \"policyId\": \"valid-type\",\n" +
+                "                                \"params\": {\n" +
+                "                                    \"types\": [\n" +
+                "                                        \"number\"\n" +
+                "                                    ]\n" +
+                "                                }\n" +
+                "                            }\n" +
+                "                        ],\n" +
+                "                        \"conditionalPolicies\": null\n" +
+                "                    }\n" +
+                "                },\n" +
+                "                {\n" +
+                "                    \"name\": \"failedPolicies\",\n" +
+                "                    \"value\": []\n" +
+                "                },\n" +
+                "                {\n" +
+                "                    \"name\": \"validateOnly\",\n" +
+                "                    \"value\": false\n" +
+                "                },\n" +
+                "                {\n" +
+                "                    \"name\": \"value\",\n" +
+                "                    \"value\": 123\n" +
+                "                }\n" +
+                "            ],\n" +
+                "            \"input\": [\n" +
+                "                {\n" +
+                "                    \"name\": \"IDToken1\",\n" +
+                "                    \"value\": null\n" +
+                "                },\n" +
+                "                {\n" +
+                "                    \"name\": \"IDToken1validateOnly\",\n" +
+                "                    \"value\": false\n" +
+                "                }\n" +
+                "            ]\n" +
+                "        }");
+        NumberAttributeInputCallback callback = new NumberAttributeInputCallback(raw, 0);
+        Assertions.assertThat(callback.getValue()).isEqualTo(123);
+    }
+}
```

Comment:
Do you think is valid to include a unit test to test a Double as input? I could not find the specification for this callback on the server side, but what is expected if a Float value is provided, should it return null as it is now? Are Double and Integer the only ones supported by this callback on the server?

Reply:
A test case for double input is added.